v2/samples/containerservice/v1api20230701/refs/v1api20210401_storageaccount.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: storage.azure.com/v1api20210401
+kind: StorageAccount
```

Comment:
Don't think this ref and below (Workspaces, Keyvault) are used in containerservice samples?

Reply:
I've simplified the samples.